daemon/daemon_unix.go

```diff
@@ -1779,3 +1779,16 @@ func (daemon *Daemon) setupSeccompProfile() error {
 func (daemon *Daemon) useShimV2() bool {
 	return cgroups.IsCgroup2UnifiedMode()
 }
+
+// RawSysInfo returns *sysinfo.SysInfo .
+func (daemon *Daemon) RawSysInfo(quiet bool) *sysinfo.SysInfo {
```

Comment:
I'm not sure I understand the need for this.
If we are adding options to the wrapped function call, why not add the operations in `RawSysInfo` as an option that callers can add when neccessary?

Reply:
Because `sysinfo.New()` should be decoupled from the daemon, and should not `daemon.getCgroupDriver()` and `os.Getenv("ROOTLESSKIT_PARENT_EUID")`

Reply:
Could we expose the sysinfo option fields?

Reply:
Could you be more specific?